### ISSUE TITLE
pickx.be

### ISSUE THREAD

**Issue by freearhey** [iptv-org/epg#2506]
### Site

pickx.be

### Description

```sh
sites/pickx.be/pickx.be.test.js (10.164 s)
  ● can generate valid url

    thrown: "Exceeded timeout of 10000 ms for a test.
    Add a timeout value to this test to increase the timeout, if this is a long-running test. See https://jestjs.io/docs/api#testname-fn-timeout."

      14 | }
      15 |
    > 16 | it('can generate valid url', async () => {
         | ^
      17 |   await fetchApiVersion()
      18 |   const generatedUrl = await url({ channel, date })
      19 |   const resolvedApiVersion = apiVersion()

      at Object.<anonymous> (sites/pickx.be/pickx.be.test.js:16:1)
```

One solution might be to mock a `fetchApiVersion` request in the test.

**Comment by BellezaEmporium** [comment]
```
@BellezaEmporium ➜ /workspaces/epg (freearhey-patch1) $ npm test --- pickx.be

> test
> run-script-os pickx.be


> test:default
> TZ=Pacific/Nauru npx jest --runInBand pickx.be

 PASS  sites/pickx.be/pickx.be.test.js (5.417 s)
  ✓ can generate valid url (2019 ms)
  ✓ can generate valid request headers (1 ms)
  ✓ can parse response (1 ms)
  ✓ can handle empty guide

Test Suites: 1 passed, 1 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        5.49 s
Ran all test suites matching /pickx.be/i.
```

I will have to check that separately.

**Comment by freearhey** [comment]
I have the test sometimes manage to pass in the allotted time (10 seconds) too:

```sh

npm test --- pickx.be

> test
> run-script-os pickx.be


> test:default
> TZ=Pacific/Nauru npx jest --runInBand pickx.be

 PASS  sites/pickx.be/pickx.be.test.js (8.391 s)
  ✓ can generate valid url (5317 ms)
  ✓ can generate valid request headers (2 ms)
  ✓ can parse response (3 ms)
  ✓ can handle empty guide (1 ms)

Test Suites: 1 passed, 1 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        8.617 s, estimated 11 s
Ran all test suites matching /pickx.be/i.

```

But to make sure that this error does not appear again, it is necessary to mock the return value of the `fetchApiVersion()` function (https://jestjs.io/docs/mock-functions#mock-return-values).

**Comment by BellezaEmporium** [comment]
I've made a basic "mock" of it, by setting the API version. It seems to pass on every part.

... and they've changed their API... what a coincidence.

**Comment by PopeyeTheSai10r** [comment]
Fixed by #2525
